.github/workflows/release.yml

```diff
@@ -212,3 +212,65 @@ jobs:
           body: |
             This is a "canary" release of the most recent commits on our main branch. Canary is **not stable**.
             It is only intended for developers wishing to try out the latest features in Spin, some of which may not be fully implemented.
+
+  create-go-sdk-tag:
+    name: create tag sdk/go/v*
+    runs-on: ubuntu-latest
+    needs: build
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set the tag sdk/go/v*
+        shell: bash
+        run: echo "GO_SDK_TAG=sdk/go/${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Tag sdk/go/v* and push it
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git tag ${{ env.GO_SDK_TAG }}
+          git push origin ${{ env.GO_SDK_TAG }} -f
+
+  commit-and-create-template-tag:
+    name: Change sdk version in templates and create a new tag spin/templates/v*
+    runs-on: ubuntu-latest
+    needs: create-go-sdk-tag
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set the tag spin/templates/v*
+        shell: bash
+        run: echo "TEMPLATE_TAG=spin/templates/${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: set the spin tag
+        shell: bash
+        run: echo "SPIN_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Change sdk version
+        shell: bash
+        run: |
+          cd templates
+          SDK_VERSION=${{ env.SPIN_TAG }} make
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v5
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "feat(templates): update sdk to ${{ env.SPIN_TAG }}"
+          commit_options: '-s -S'
+          commit_user_name: fermybot
+          commit_user_email: bot@fermyon.com
```

Comment:
We can use `103076628+fermybot@users.noreply.github.com` for both emails

Reply:
Updated it. Thank you.